api/src/pcapi/core/finance/backend/cegid.py

```diff
@@ -243,21 +246,24 @@ def push_invoice(self, invoice: finance_models.Invoice) -> dict:
             response.status_code == 500
             and self._get_exception_type(response) == "PX.Api.ContractBased.OutcomeEntityHasErrorsException"
         ):
-            raise exceptions.FinanceBackendInvoiceAlreadyExists(
-                response, f"Invoice '{invoice.reference}' already pushed"
-            )
+            # If the invoice is already pushed, we might be in a retry due to a timeout from the first PUT query
+            # We thus get the id necessary for the POST
+            logger.warning(f"Invoice '{invoice.reference}' already pushed")
```

Comment:
Il me semble qu'on ne doit pas utiliser de f-string pour les logs ; pas d'alerte ? Était-ce pylint qui râlait, mais ruff plus tolérant ?
```suggestion
            logger.warning("Invoice '%s' already pushed", invoice.reference)
```